docs/index.md

```diff
@@ -26,16 +26,30 @@ trick enemies by having them see you in one position and you moving up behind th
 ## Dynamic map creation?
 To enhance replayability, we developed a system to dynamically generate maps in our game. One of the requierments of the
 project was to be able to create maps from strings, so a logical step to this was to first dynamically create the strings
-then create the map based on said strings. We settled on a three method aproach, where the TileMap-class was responsible
+then create the map based on said strings. We settled on a three method approach, where the TileMap-class was responsible
 for converting the StringMap-objects into tiles that could be used by the gamehandler. The StringMap-class was responsible
 for putting together pre-made sections of maps into a full mapString. The pre-made sections were divided into either named
--rooms or numbered-rooms. The funtion of named-rooms was to ensure we always had a spawn location for the player, and a way
+-rooms or numbered-rooms. The function of named-rooms was to ensure we always had a spawn location for the player, and a way
 to get to the next map. It also allows for later adding specialised rooms, such as bosses, but we ended up only expanding this
 function to include test-maps. The numbered-rooms were sections intended for random selection, and a StringMap would always
 consist of a startingRoom, an endRoom and a number of randomly selected numbered-rooms. In order for the StringMap-class to
 know where to place the rooms however, it needed a GridMap-object. The GridMap-class created a Grid, always starting with the
 spawn-room of the player, and then randomly picked a direction (up, down, left, right) and placed a random room. Then it continued
-this behaviour, randomly picking a location adjacent to a previously placed room untill it had placed all the random- rooms before
+this behaviour, randomly picking a location adjacent to a previously placed room until it had placed all the random- rooms before
```

Comment:
I guess it should be `random rooms` without the hyphen. Optional to fix

Reply:
Fixed!